pkg/storage/pvc.go

```diff
@@ -315,3 +344,17 @@ func (builder *PVCBuilder) validate() (bool, error) {
 
 	return true, nil
 }
+
+// validateVolumeMode validates if requested volume mode is valid for PVC.
+func validateVolumeMode(volumeMode string) bool {
+	glog.V(100).Info("Validating volumeMode %s", volumeMode)
+
+	var validVolumeModes = map[string]string{
```

Comment:
do we need to create a dict for 2 possible string values? i.e. why not just comparing to one of the strings?

Reply:
Or just use the slices.Contains() function. Example - https://github.com/openshift-kni/eco-goinfra/blob/main/pkg/rbac/clusterrolebinding.go#L85

Reply:
thanks, updated